dom/events/EventListener-handleEvent.html

```diff
@@ -66,6 +66,19 @@
     assert_equals(calls, 2);
 }, "performs `Get` every time event is dispatched");
 
+test(function(t) {
+    var type = "foo";
+    var target = document.getElementById("target");
+
+    var eventListener = function() {};
```

Comment:
Add something here that proves the function itself is called?

Reply:
d2a45d1